crowd_anki/config/config_dialog.py

```diff
@@ -32,6 +32,7 @@ def accept(self):
             super().accept()
 
     def ui_initial_setup(self):
+        # Snapshot Options
```

Comment:
better just extract each of those sections into a separate method

Reply:
Done :+1: